src/main/java/seedu/address/model/tag/TagCategory.java

```diff
@@ -6,7 +6,7 @@
  * Provides categories to sort Tags into, as well as colour codes for display in the UI.
  */
 public enum TagCategory {
-    GENERAL("#ECECEC"), // Light Grey for default color
+    GENERAL("#A9A9A9"), // Dark Grey for default color
```

Comment:
Not sure if dark grey is a good colour because it could mean "invalid tag" to a user that's not familiar to our codebase, but I think it suffices for now